core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -228,7 +230,8 @@ export class ExplorationEditorPageComponent implements OnInit, OnDestroy {
       this.userService.getUserInfoAsync()
     ]).then(async(
         [explorationData, featuresData, _, userInfo]) => {
-      if ((explorationData as ExplorationData).exploration_is_linked_to_story) {
+      if ((explorationData as unknown as ExplorationData)
```

Comment:
here and elsewhere ?

Reply:
@soumyo123-prog Please do not use `as unknown`.

Reply:
Removed it.